server/src/api_endpoints/personas/persona.js

```diff
@@ -82,10 +82,7 @@ const getPersona = async (req, res, next) => {
     const { persona_key: key } = req.params
     const [persona] = await dbAdminPersona.getPersonas({ key })
     if (!persona) {
-      throw {
```

Comment:
The `throw` is being caught in the `catch` block. Line 92

Reply:
@k1ch  yea, I saw that while reviewing, but I was thinking that since we know the exact error we can call next right away. 
This is related to the larger discussion in #112 .